utils/Makefile

```diff
@@ -33,10 +32,6 @@ MANPAGES := docs/man/man1/atomic-openshift-installer.1
 # slipped into the manpage template before a2x processing
 VERSION := 1.4
 
-# YAMLFILES: Skipping all '/files/' folders due to conflicting yaml file definitions
-YAMLFILES = $(shell find ../ -name $(VENV) -prune -o -name .tox -prune -o \( -name '*.yml' -o -name '*.yaml' \) ! -path "*/files/*" -print 2>&1)
-PYFILES = $(shell find ../ -name $(VENV) -prune -o -name ooinstall.egg-info -prune -o -name test -prune -o -name .tox -prune -o -name "*.py" -print)
```

Comment:
These are no longer used, neither here not anywhere else. Our test runner / discovery is based on tox/pytest.